website/docs/r/variable.html.markdown

```diff
@@ -91,6 +91,46 @@ in the state.
 ## Attributes Reference
 
 * `id` - The ID of the variable.
+* `readable_value` - Only present if the variable is non-sensitive. A copy of the value which will not be marked as sensitive in plan outputs.
+
+### Using readable_value
+
+While the `value` field may be referenced in other resources, for safety it is always treated as sensitive. This means that it will always be redacted from plan outputs, and any other resource attributes which depend on it will also be redacted.
+
+The `readable_value` attribute is not sensitive, and will not be redacted. This allows other resources to reference it, while keeping their plan outputs readable.
```

Comment:
```suggestion
The `readable_value` attribute is not sensitive, and will not be redacted; instead, it will be null if the variable is sensitive. This allows other resources to reference it, while keeping their plan outputs readable.
```

Well, idk, that one counts as "take it or leave it," since you say it enough times elsewhere in the docs. 😅 I'm just role-playing as a paranoid who stops reading early and freaks out a bit.